sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -28,56 +28,6 @@ Use the client library for App Configuration to create and manage application co
 ```
 [//]: # ({x-version-update-end})
 
-### Default HTTP Client
-All client libraries, by default, use Netty HTTP client. Adding the above dependency will automatically configure 
-AppConfiguration to use Netty HTTP client. 
-
-### Alternate HTTP Client
-If, instead of Netty it is preferable to use OkHTTP, there is a HTTP client available for that too. Exclude the default
-Netty and include OkHTTP client in your pom.xml.
-
-[//]: # ({x-version-update-start;com.azure:azure-data-appconfiguration;current})
-```xml
-<!-- Add AppConfiguration dependency without Netty HTTP client -->
-<dependency>
-    <groupId>com.azure</groupId>
-    <artifactId>azure-data-appconfiguration</artifactId>
-    <version>1.0.1</version>
-    <exclusions>
-      <exclusion>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-core-http-netty</artifactId>
-      </exclusion>
-    </exclusions>
-</dependency>
-```
-[//]: # ({x-version-update-start;com.azure:azure-core-http-okhttp;current})
-```xml
-<!-- Add OkHTTP client to use with AppConfiguration -->
-<dependency>
-  <groupId>com.azure</groupId>
-  <artifactId>azure-core-http-okhttp</artifactId>
-  <version>1.1.0</version>
-</dependency>
-```
-[//]: # ({x-version-update-end})
-
-### Configuring HTTP Clients
-When an HTTP client is included on the classpath, as shown above, it is not necessary to specify it in the client library [builders](#create-client), unless you want to customize the HTTP client in some fashion. If this is desired, the `httpClient` builder method is often available to achieve just this, by allowing users to provide a custom (or customized) `com.azure.core.http.HttpClient` instances.
```

Comment:
Also, cleanup `#create-client` link at bottom of the README if it's no longer used. Same for other links that were deleted.

Reply:
the link is no longer exists in the first place. Should remove it.

Reply:
We can keep create an azconfig client section for other use.